datashuttle/utils/gdrive.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+from datashuttle.configs.config_class import Configs
+from datashuttle.utils import rclone, utils
+
+
+def preliminary_for_setup_without_browser(
+    cfg: Configs, rclone_config_name: str, log: bool = True
+):
+    client_id_key_value = (
+        f"client_id {cfg['gdrive_client_id']} "
+        if cfg.get("gdrive_client_id", None)
+        else " "
+    )
+    client_secret_key_value = (
+        f"client_secret {cfg['gdrive_client_secret']} "
+        if cfg.get("gdrive_client_secret", None)
+        else ""
+    )
+    output = rclone.call_rclone(
```

Comment:
Some of these arguments to the rclone call are a little mysterious, a short comment above would be useful just to explain to readers what is going on. Of course, its always possible to check the rclone docs, but in general it is more efficient to leave a short comment to give the pertinent details.